examples/language-modeling/main.py

```diff
@@ -1,18 +1,19 @@
-import argparse
+import os
+import re
 import sys
+import argparse
+import subprocess
+from packaging import version
 
 sys.path.insert(0, '../..')
 parser = argparse.ArgumentParser()
 import torch
-import os
 import transformers
-import subprocess
 
 os.environ["CUBLAS_WORKSPACE_CONFIG"] = ":4096:8"
 torch.use_deterministic_algorithms(True, warn_only=True)
 from transformers import AutoModelForCausalLM, AutoTokenizer, AutoModel
 
-import re
 
 os.environ["TOKENIZERS_PARALLELISM"] = "false"
```

Comment:
please change the lm-eval in the examples/language-modeling/requirement to 0.4.3